### PR TITLE
eap-cluster-app README polishing

### PR DESCRIPTION
**Pull request by Holmistr**
Fixed some formatting error, fixed some typos, just to make the README look pretty.

### CODE REVIEW

File: eap-cluster-app/README.md [hunk 1]
```diff
@@ -48,16 +48,16 @@ Configure and Start the Servers in standalone mode
 --------------------------------------------------
 
 1. Prepare a copy of the JBoss EAP
-   - unzip jboss-datagrid-${version}-eap-modules-library.zip where ${version} is 6.3.1 or better
+   - unzip `jboss-datagrid-${version}-eap-modules-library.zip` where `${version}` is 6.4 or better
```
Comment (slaskawi) [inline eap-cluster-app/README.md]: ...is 6.4+ would be consistent.

File: eap-cluster-app/README.md [hunk 2]
```diff
@@ -83,11 +83,11 @@ Configure and Start the Servers in domain mode
 ----------------------------------------------
 
 1. Copy a fresh JBoss EAP installation to EAP_HOME
-   - unzip jboss-datagrid-${version}-eap-modules-library.zip where ${version} is 6.3.1 or better
+   - unzip `jboss-datagrid-${version}-eap-modules-library.zip` where ${version} is 6.4 or better
```
Comment (slaskawi) [inline eap-cluster-app/README.md]: The same here

**Comment by slaskawi** [comment]
Looks great! Only 2 small findings...

**Comment by Holmistr** [comment]
@slaskawi Comments addressed and repushed

**Comment by slaskawi** [comment]
Looks great! Thanks!

**Pull request merged**